packages/edit-site/src/components/sidebar-edit-mode/index.js

```diff
@@ -69,7 +69,7 @@ export function SidebarComplementaryAreaFills() {
 	// See https://github.com/WordPress/gutenberg/blob/trunk/docs/how-to-guides/feature-flags.md#dead-code-elimination.
 	let MaybeNavigationMenuSidebar = Fragment;
 
-	if ( process.env.IS_GUTENBERG_PLUGIN ) {
+	if ( window?.__experimentalEnableOffCanvasNavigationEditor === true ) {
```

Comment:
Non-blocking note: other (most?) experimental flags are assigned to a variable before use.
E.g.
```js
const isOffCanvasNavigationEditorEnabled =
	window?.__experimentalEnableOffCanvasNavigationEditor === true;

if ( isOffCanvasNavigationEditorEnabled ) {
  // ...
}
```

The only other occurrence I could find of the global used directly is another off-canvas nav flag:

https://github.com/WordPress/gutenberg/blob/c05d7c5535a38ae18661c8c7e99c1aec72720722/packages/edit-site/src/components/sidebar-edit-mode/navigation-menu-sidebar/navigation-menu.js#L53

Whatever, not a big deal, IMHO. 🙂